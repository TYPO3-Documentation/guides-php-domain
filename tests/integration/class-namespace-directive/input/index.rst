===============================================
PHP Class with current namespace from directive
===============================================

..  php:namespace:: TYPO3\CMS\Core

..  php:class:: TestClass

    Lorem Ipsum Dolor!

..  php:class:: AnotherClass

    Lorem Ipsum Dolor Another!

..  php:class:: MyVendor\Some\Namespace\AnotherClass

    Lorem Ipsum Dolor Yet Another!
