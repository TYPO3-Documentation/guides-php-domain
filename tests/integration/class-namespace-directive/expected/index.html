<!-- content start -->
    <div class="section" id="php-class-with-current-namespace-from-directive">
            <h1>PHP Class with current namespace from directive</h1>
            <dl class="php class">
    <dt class="sig sig-object php" id="typo3-cms-core-testclass">
        <em class="property"><span class="pre">class</span> </em>
        <span class="sig-name descname">
    <span class="pre fqn" data-namespace="\TYPO3\CMS\Core" title="\TYPO3\CMS\Core\TestClass">
        TestClass
    </span>
</span> <a class="headerlink" href="#typo3-cms-core-testclass" title="Permalink to this definition">¶</a>
    </dt>
    <dd>
                    <dl class="field-list simple">
                                    <dt class="field-even">Fully qualified name</dt>
                    <dd class="field-even">
                                                <span class="pre">\TYPO3\CMS\Core\TYPO3\CMS\Core</span>                    </dd>
                </dl>

    <p>Lorem Ipsum Dolor!</p>

    </dd>
</dl>
            <dl class="php class">
    <dt class="sig sig-object php" id="typo3-cms-core-anotherclass">
        <em class="property"><span class="pre">class</span> </em>
        <span class="sig-name descname">
    <span class="pre fqn" data-namespace="\TYPO3\CMS\Core" title="\TYPO3\CMS\Core\AnotherClass">
        AnotherClass
    </span>
</span> <a class="headerlink" href="#typo3-cms-core-anotherclass" title="Permalink to this definition">¶</a>
    </dt>
    <dd>
                    <dl class="field-list simple">
                                    <dt class="field-even">Fully qualified name</dt>
                    <dd class="field-even">
                                                <span class="pre">\TYPO3\CMS\Core\TYPO3\CMS\Core</span>                    </dd>
                </dl>

    <p>Lorem Ipsum Dolor Another!</p>

    </dd>
</dl>
            <dl class="php class">
    <dt class="sig sig-object php" id="myvendor-some-namespace-anotherclass">
        <em class="property"><span class="pre">class</span> </em>
        <span class="sig-name descname">
    <span class="pre fqn" data-namespace="\MyVendor\Some\Namespace" title="\MyVendor\Some\Namespace\AnotherClass">
        AnotherClass
    </span>
</span> <a class="headerlink" href="#myvendor-some-namespace-anotherclass" title="Permalink to this definition">¶</a>
    </dt>
    <dd>
                    <dl class="field-list simple">
                                    <dt class="field-even">Fully qualified name</dt>
                    <dd class="field-even">
                                                <span class="pre">\MyVendor\Some\Namespace\MyVendor\Some\Namespace</span>                    </dd>
                </dl>

    <p>Lorem Ipsum Dolor Yet Another!</p>

    </dd>
</dl>
    </div>
<!-- content end -->
